doc/client/play.md

```diff
@@ -0,0 +1,60 @@
+# Using as a Play client
+
+Add the dependency:
+
+```scala
+"com.softwaremill.sttp.tapir" %% "tapir-play-client" % "@VERSION@"
+```
+
+To make requests using an endpoint definition using the [play client](https://github.com/playframework/play-ws), import:
+
+```scala mdoc:compile-only
+import sttp.tapir.client.play._
+```
+
+This adds the two extension methods to any `Endpoint`:
+ - `toPlayRequestUnsafe(String)`: given the base URI returns a function,
+   which will generate a request and a response parser which might throw
+   an exception when decoding of the result fails
+   ```scala
+   I => (StandaloneWSRequest, StandaloneWSResponse => Either[E, O])
+   ```
+ - `toPlayRequest(String)`: given the base URI returns a function,
+   which will generate a request and a response parser which represents
+   decoding errors as the `DecodeResult` class
+   ```scala
+   I => (StandaloneWSRequest, StandaloneWSResponse => DecodeResult[Either[E, O]])
+   ```
+
+Note that these are a one-argument functions, where the single argument is the input of end endpoint. This might be a 
+single type, a tuple, or a case class, depending on the endpoint description. 
+
+After providing the input parameters, the two following are returned:
+- a description of the request to be made, with the input value
+  encoded as appropriate request parameters: path, query, headers and body.
+  This can be further customised and sent using regular Play methods.
+- a response parser to be applied to the response got after executing the request.
+  The result will then contain the decoded error or success values
+  (note that this can be the body enriched with data from headers/status code).
+
+Example:
+
+```scala
```

Comment:
I'd try adding `scala mdoc:compile-only` to that this example is compiled (when running `mdoc` in the documentation project)

Reply:
Sure, I'm not familiar with `mdoc` but it's an opportunity to see how it works :)

I'll do it in another PR as I see you've already merged this one.

Reply:
I've fixe dthis already :)

Reply:
https://github.com/softwaremill/tapir/commit/4d56e0ef26aac2a6099e0b16a121ebd6c2ffdbae

Reply:
Oh great thanks :)